da/da.go

```diff
@@ -95,11 +95,26 @@ type ResultRetrieveBlocks struct {
 
 // DAClient is a new DA implementation.
 type DAClient struct {
-	DA            goDA.DA
-	GasPrice      float64
-	GasMultiplier float64
-	Namespace     goDA.Namespace
-	Logger        log.Logger
+	DA              goDA.DA
+	GasPrice        float64
+	GasMultiplier   float64
+	Namespace       goDA.Namespace
+	SubmitTimeout   time.Duration
+	RetrieveTimeout time.Duration
+	Logger          log.Logger
+}
+
+// NewDAClient returns a new DA client.
+func NewDAClient(da goDA.DA, gasPrice, gasMultiplier float64, ns goDA.Namespace, logger log.Logger) *DAClient {
+	return &DAClient{
+		DA:              da,
+		GasPrice:        gasPrice,
+		GasMultiplier:   gasMultiplier,
+		Namespace:       ns,
+		SubmitTimeout:   defaultSubmitTimeout,
```

Comment:
`SubmitTimeout` and `RetrieveTimeout` are not set-able, why even add to struct and set using defaults?

Reply:
It's being used to test submit timeout, as per: https://github.com/rollkit/rollkit/pull/1587#discussion_r1523937821